### PR TITLE
Update Postgres extensions

### PR DESCRIPTION
**Pull request by bayandin**
## Describe your changes
- Update postgis to 3.3.2 from 3.3.1
- Update plv8 to 3.1.5 from 3.1.4
- Update h3-pg to 4.1.2 from 4.0.1
  - Update h3 to 4.1.0 from 4.0.1
## Issue ticket number and link
N/A

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [x] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

### CODE REVIEW

File: Dockerfile.compute-node [hunk 1]
```diff
@@ -50,15 +50,14 @@ RUN apt update && \
     libcgal-dev libgdal-dev libgmp-dev libmpfr-dev libopenscenegraph-dev libprotobuf-c-dev \
     protobuf-c-compiler xsltproc
 
-RUN wget https://gitlab.com/Oslandia/SFCGAL/-/archive/v1.3.10/SFCGAL-v1.3.10.tar.gz && \
```
Comment (vadim2404) [inline Dockerfile.compute-node:53]: Look at the last step. Finally, we're installing this lib via `apt-get` for the final image. We must be compatible. Have you tested it?

File: Dockerfile.compute-node [hunk 2]
```diff
@@ -98,12 +97,9 @@ RUN wget https://ftp.gnu.org/gnu/binutils/binutils-2.38.tar.gz && \
...
     cp /usr/local/bin/ld.gold /usr/bin/gold
 
-# Sed is used to patch for https://github.com/plv8/plv8/issues/503
-RUN wget https://github.com/plv8/plv8/archive/refs/tags/v3.1.4.tar.gz && \
-    tar xvzf v3.1.4.tar.gz && \
-    cd plv8-3.1.4 && \
+RUN wget https://github.com/plv8/plv8/archive/refs/tags/v3.1.5.tar.gz -O plv8.tar.gz && \
+    mkdir plv8 && cd plv8 && tar xvzf ../plv8.tar.gz --strip-components=1 -C . && \
     export PATH="/usr/local/pgsql/bin:$PATH" && \
-    sed -i 's/MemoryContextAlloc(/MemoryContextAllocZero(/' plv8.cc && \
```
Comment (vadim2404) [inline Dockerfile.compute-node:106]: https://github.com/plv8/plv8/pull/504/files

I hope you relied on this PR

File: Dockerfile.compute-node (hunk 1 above)
Comment (bayandin) [reply Dockerfile.compute-node:53]: Added a comment about it

File: Dockerfile.compute-node (hunk 2 above)
Comment (bayandin) [reply Dockerfile.compute-node:106]: Yup, it's merged into v3.1.5: https://github.com/plv8/plv8/commit/d38ab55a93e142185486cc7982955b629045390c

**Pull request merged**